<?xml version = "1.0"?>

<launch>

    <!-- <arg name="marker_size" default="15.0" />
    <arg name="max_new_marker_error" default="0.01" />
    <arg name="max_track_error" default="0.05" />

    <node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen">
        <param name="marker_size"           type="double" value="$(arg marker_size)" />
        <param name="max_new_marker_error"  type="double" value="$(arg max_new_marker_error)" />
        <param name="max_track_error"       type="double" value="$(arg max_track_error)" />
        <param name="output_frame"          type="string" value="camera_frame" />

        <remap from="/camera_image"  to="/zed2/left/image_rect_color" />
        <remap from="/camera_info"   to="/zed2/left/camera_info" />
    </node> -->

    <node name="poseDetection" pkg="rover" type="ar_track_pose" respawn="false" output="screen"/>
    <!-- <node name="poseDetection" pkg="rover" type="poseDetection" respawn="false" output="screen"/> -->

</launch>